src/Components/Shared/ModalAlert/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import styles from './modal-alert.module.css';
+const ModalAlert = (props) => {
+  return (
+    <div className={styles.modal}>
+      <div className={styles['modal-content']}>
+        <p>{props.texto}</p>
+        <button onClick={props.onConfirmar}>Confirm</button>
```

Comment:
should use the shared comoponent "button"

Reply:
implemented